### ISSUE TITLE
Executing npx command given by the tool doesn't work

### ISSUE THREAD

**Issue by m4rcoperuano** [formkit/theme-starter#1]
Hello! This could be just me, but when running this command that is outputted by the tool:

```
npx formkit@latest theme --theme=regenesis --variables="radius=rounded,accentColor=emerald,colorTemperature=neutral,spacing=2,scale=base,inputShadow=shadow,baseColorShade=600,borderShadeLightMode=400,borderShadeDarkMode=500,inputMaxWidth=max-w-[20em],tagRadius=rounded,swatchRadius=rounded"
error: unknown command 'theme'
```

I get an error saying `error: unknown command 'theme'`. 

Not sure if its maybe my npm installation or... something else. Reproducable by just going to this url: https://themes.formkit.com/editor?theme=regenesis&variables=radius%3Drounded,accentColor%3Dblue,colorTemperature%3Dneutral,spacing%3D2,scale%3Dbase,inputShadow%3Dshadow,baseColorShade%3D600,borderShadeLightMode%3D400,borderShadeDarkMode%3D500,inputMaxWidth%3Dmax-w-%5B20em%5D,tagRadius%3Drounded,swatchRadius%3Drounded, clicking Download Theme, copying the CLI code, and pasting into your terminal. 

I can confirm that I am in my root directory of my application that already has formkit working. I just wanted to change its style. This is a Laravel app running Intertia (Vue 3).

**Comment by justin-schroeder** [comment]
Sounds like your FormKit command is not the latest? What happens if you run `npx formkit —version`? If that isn’t 1.4.0 try globally updating your formkit package.

**Comment by m4rcoperuano** [comment]
Thank you guys. Fixed! Here's what happened:
1. This project has been going on for a few months. When we first imported formkit, it was formkit v1.1.0. 
2. I wanted to try out the pro components of formkit, so i followed the guide to do so.
3. I noticed that my component styles were missing, so I eventually found my way into the new themes site.
4. I wasn't aware that the themes update came with version 1.4.0, so that was my bad. 
5. After i noticed i was running v1.1.0, i upgraded to v1.4.0.
6. Then I ran the command again and got another error, had something to do with node not being able to execute fetch (or await). 
7. This led to me seeing that I was running node 16. Upgraded to Node 18, and now it worked :)

But i wouldn't have been able to figure this out as quickly without that `npx formkit --version` command. Thanks guys for the tips!
